scripts/date.js

```diff
@@ -0,0 +1,9 @@
+const today = new Date();
+const date = String(today.getDate())
+const month = new Intl.DateTimeFormat('en-US', { month: 'long'}).format(today)
+const day = new Intl.DateTimeFormat('en-US',{ weekday: 'long' }).format(today)
+const year = today.getFullYear();
+
+const displayDate = `${day} ${month} ${date}, ${year}`;
+
+document.getElementById("today-date").innerHTML = `<h3>${displayDate}</h3>`
```

Comment:
This file mixes logic and presentation. Maybe you could wrap the functionality to format today's date into a function, and then in the interface, where you set up the rest of the page, you can replace it.

Reply:
That would also avoid declaring all these constants to the global scope.

Reply:
So:
```js
function formattedTodaysDate() {
  const today = new Date();
  // ...
  return `${day} ${month} ${date}, ${year}`
}
```

Or maybe even
```js
function formattedDate(date) {
  const month = new Intl.DateTimeFormat('en-US', { month: 'long'}).format(date)
  // ...
  return `${day} ${month} ${monthDay}, ${year}`
}

// then call like this for todays'
formattedDate(new Date())
```